internal/install/execution/installevents_reporter.go

```diff
@@ -154,6 +154,16 @@ func (r InstallEventsReporter) createInstallStatusEvent(state installevents.Inst
 	return err
 }
 
+func (r InstallEventsReporter) UpdateRequired(status *InstallStatus) error {
+	err := r.createInstallStatusEvent(
+		installevents.InstallationInstallStateType(installevents.InstallationRecipeStatusTypeTypes.UNSUPPORTED),
```

Comment:
This meets the current requirements, but wondering if we need an additional state enum/scalar to better categorize this.